docs/learn-proxies.md

```diff
@@ -0,0 +1,66 @@
+---
+id: learn-proxies
+title: Proxy Accounts
+sidebar_label: Proxy Accounts
+---
+
+Polkadot provides a module that allows users to set proxy accounts to perform a limited number of
+actions on their behalf. Much like the Stash and Controller account relationship in
+[staking](learn-staking), proxies allow users to keep one account in cold storage and actively
+participate in the network with the weight of the tokens in that account.
+
+## Proxy Types
+
+You can set a proxy account via the Proxy module. When you set a proxy, you must choose a type of
+proxy for the relationship. Polkadot offers:
+
+- Any
+- Non-transfer
+- Staking
+- Governance
+
+When a proxy account makes a `proxy` transaction, Polkadot filters the desired transaction to ensure
+that the proxy account has the appropriate permission to make that transaction on behalf of the cold
+account.
+
+As implied by the name, a proxy type of "Any" allows the proxy account to make any transaction,
+incuding balance transfers. In most cases, this should be avoided as the proxy account is used more
+frequently than the cold account and is therefore less secure. To allow any type of transaction
+except balance transfers (including vested transfers), the "Non-transfer" proxy type will be a more
+apposite fit.
+
+The "Governance" type will allow proxies to make transactions related to governance (i.e., from the
+Democracy, Council, Treasury, Technical Committee, and Elections pallets).
+
+The "Staking" type allows staking-related transactions, but do not confuse a staking proxy with the
+Controller account. Within the Staking pallet, some transactions must come from the Stash, while
+others must come from the Controller.
+
+### Anonymous Proxies
+
+Polkadot includes a function to create an anonymous proxy, an account that can only be accessed via
```

Comment:
TBH I'm a bit fuzzy on their applicability. They're not really anonymous since you still have to call from the account that created them. They are more like _exclusive_ proxies. Didn't come up with a use case but just tried to explain it better.